arch/arm/src/common/arm_initialize.c

```diff
@@ -122,11 +122,9 @@ void up_initialize(void)
   arm_serialinit();
 #endif
 
-#ifndef CONFIG_NETDEV_LATEINIT
```

Comment:
arm_usbinitialize and arm_pminitialize can do the same thing.

Reply:
Yes. Will do USB clean-up in a separate PR

Reply:
That's fine in another PR.